aptos-move/aptos-gas/src/transaction/storage.rs

```diff
@@ -308,14 +308,9 @@ pub struct StorageGasParameters {
 impl StorageGasParameters {
     pub fn new(
         feature_version: u64,
-        gas_params: Option<&AptosGasParameters>,
+        gas_params: &AptosGasParameters,
         storage_gas_schedule: Option<&StorageGasSchedule>,
-    ) -> Option<Self> {
-        if feature_version == 0 || gas_params.is_none() {
```

Comment:
what happens to feature_version = 0?

Reply:
`feature_version == 0 && param_param.is_none()` is only when running the genesis, where a `UnmeteredGasMeter` is used (in fact if we don't construct the `StorageGasParameters` here, a standard gas meter will fail to construct)